ffs/manager/manager_test.go

```diff
@@ -113,32 +176,38 @@ func TestDefaultStorageConfig(t *testing.T) {
 	// Change the default config and test.
 	c.Hot.Enabled = false
 	c.Repairable = true
-	err := m.SetDefaultStorageConfig(c)
+	err = m.SetDefaultStorageConfig(c)
 	require.NoError(t, err)
 	c2 := m.GetDefaultStorageConfig()
 	require.Equal(t, c, c2)
-	cls()
+	defer require.NoError(t, cls())
 
 	// Re-open manager, and check that
 	// the default config was the last
 	// saved one, and not zeroConfig.
-	m, cls = newManager(t, ds)
-	defer cls()
+	m, cls, err = newManager(client, ds, addr, false)
+	require.NoError(t, err)
+	defer require.NoError(t, cls())
 	c3 := m.GetDefaultStorageConfig()
 	require.Equal(t, c, c3)
 }
 
-func newManager(t *testing.T, ds datastore.TxnDatastore) (*Manager, func()) {
-	client, addr, _ := tests.CreateLocalDevnet(t, 1)
-	wm, err := walletModule.New(client, addr, *big.NewInt(4000000000), false, "")
-	require.NoError(t, err)
+func newManager(client *apistruct.FullNodeStruct, ds datastore.TxnDatastore, masterAddr address.Address, ffsUseMasterAddr bool) (*Manager, func() error, error) {
```

Comment:
Kind of annoying refactor here, but needed to do it to be able to test that creating the manager fails in certain cases.